runtime/near-vm-logic/src/logic.rs

```diff
@@ -1268,6 +1266,14 @@ impl<'a> VMLogic<'a> {
         self.checked_push_promise(Promise::Receipt(new_receipt_idx))
     }
 
+    /// Helper function to return the account id towards which the receipt is directed.
+    fn get_account_by_receipt(&self, receipt_idx: &ReceiptIndex) -> AccountId {
+        self.receipt_to_account
+            .get(receipt_idx)
+            .expect("promises and receipt_to_account should be consistent.")
+            .clone()
```

Comment:
why do you need to clone here?

Reply:
I thought about returning `AccountId` without reference to simplify code at some place. It didn't help, but I forgot to revert this change.